client/components/common/tce-core/AddElement/index.vue

```diff
@@ -17,32 +17,42 @@
         <v-icon>{{ icon }}</v-icon>
       </v-btn>
     </slot>
-    <v-bottom-sheet v-model="isVisible" max-width="1240" inset>
+    <v-bottom-sheet
+      @input="isVisible = $event"
+      :value="isVisible && !showElementBrowser"
+      max-width="1240" inset>
       <div class="element-container">
         <v-toolbar
-          v-if="layout"
           color="blue-grey darken-4"
           dense
           class="mb-2 elevation-1">
-          <v-spacer />
-          <v-divider vertical />
-          <v-btn-toggle
-            v-model="elementWidth"
-            active-class="blue-grey darken-2"
-            background-color="transparent"
-            dark tile borderless mandatory>
-            <v-btn :value="100" icon>
-              <v-icon>mdi-square-outline</v-icon>
-            </v-btn>
-            <v-btn :value="50" icon>
-              <v-icon>mdi-select-compare</v-icon>
-            </v-btn>
-          </v-btn-toggle>
-          <v-divider class="mr-3" vertical />
-          <div class="width-label px-1 subtitle-1 grey--text text--lighten-4">
-            Element width
-            <span class="px-1">{{ elementWidth }}</span>%
-          </div>
+          <v-btn
```

Comment:
Does it make sense to create a slot and control add element visibility from outside (Move this component into AddNewElement and have a new index.vue wrapper that does add new / add existing handling)? ` :value="isVisible && !showElementBrowser"` looks a bit off 🤔

Reply:
Craves for some sort of separation 😄 , but I am kind of running in circles trying to clean this up... Have some ideas but I would like to resolve that in a separate PR. For now, I just moved some template and styles to `AddNewElement` component.